code/game/objects/items/weapons/circuitboards/machinery/network.dm

```diff
@@ -106,4 +106,23 @@
 
 	additional_spawn_components = list(
 		/obj/item/stock_parts/power/apc/buildable = 1
-	)
\ No newline at end of file
+	)
+
+
+/obj/item/stock_parts/circuitboard/trade_controller
+	name = "circuitboard (trade control device)"
+	build_path = /obj/machinery/network/trade_controller
+	origin_tech = "{'programming':4,'magnets':3}"
+	req_components = list(
+		/obj/item/stock_parts/capacitor = 1,
+		/obj/item/stock_parts/scanning_module = 2
+		)
+
+	additional_spawn_components = list(
+		/obj/item/stock_parts/power/apc/buildable = 1
```

Comment:
You can delete that line, it puts an apc in by default in the base circuit board.

Reply:
I've made the change, although the banking mainframe it's copied from directly above still has the line.

Reply:
Yeah, don't worry about it. We'll eventually get to fixing that other one too.